parameter/DomainConfiguration.cpp

```diff
@@ -488,8 +498,12 @@ void CDomainConfiguration::split(CConfigurableElement* pFromConfigurableElement)
         // Get child configurable elemnt's area configuration
         CAreaConfiguration* pChildAreaConfiguration = getAreaConfiguration(pToChildConfigurableElement);
 
-        // Do the copy
-        pChildAreaConfiguration->copyFromOuter(pAreaConfigurationToSplitFrom);
+        if (pChildAreaConfiguration != NULL) {
+            // Do the copy
+            pChildAreaConfiguration->copyFromOuter(pAreaConfigurationToSplitFrom);
+        } else {
+            assert(pChildAreaConfiguration);
+        }
```

Comment:
As far as y know, replacing the assert by abort as you did should remove the need for those `pChildAreaConfiguration != NULL` tests. Could you confirm/infirm ?